### PR TITLE
Integration tests for pathfinder

### PR DESCRIPTION
**Pull request by anderslime**

### CODE REVIEW

File: test/integration/pathfinder_test.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  test "resource routes" do
+    assert_includes @static_routes, "/locations"
+  end
+
+  test "wildcard routes" do
+    assert_includes @wildcard_routes, "/wildcard/*/route"
+  end
+end
+
```
Comment (omegahm) [inline test/integration/pathfinder_test.rb]: :space_invader:

**Comment by omegahm** [comment]
Are you setting it up on Jenkins as well? So that it will run these specs?

**Comment by anderslime** [comment]
| Are you setting it up on Jenkins as well? So that it will run these specs?

 Jep. I will close this PR and continue from #5

**Comment by anderslime** [comment]
Jenkins, do your stuff

**Pull request closed without merging**